package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.4.5",
   "private": true,
   "scripts": {
-    "start": "NETWORK=regtest webpack --watch --config webpack.dev.js",
+    "start": "NETWORK=regtest webpack --config webpack.dev.js",
```

Comment:
Why remove watch?

Reply:
Better to put it in the config file I think (and it lets us to set up the "watchOptions" field also)